langs/jq/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.20 AS builder
+
+ARG RELEASE=1.7.1
+
+RUN apk add --no-cache build-base curl
+
+RUN curl -L https://github.com/jqlang/jq/releases/download/jq-$RELEASE/jq-$RELEASE.tar.gz | tar xz
+
+WORKDIR /jq-$RELEASE
+
+RUN ./configure --with-oniguruma=builtin \
+ && make -j8 && make check && make install
```

Comment:
Use
```
-j`nproc`
```
to avoid making assumptions about how many CPUs I have :-P

Reply:
Huh?! I usually already do.